docs/application/native/guides/text-input/autofill-service.md

```diff
@@ -0,0 +1,335 @@
+# Autofill Service
+
+Autofill Service application is a service application to store the user input and provides the data previously entered by the user. Developers can implement the Autofill service in their own application to serve the Autofill information.
+
+The main features of the Autofill service API include:
+
+- Authentication Information
+
+  You can [receive the request of authentication, and send the authentication information](#authentication-information).
+
+- Autofill fill request and autofill fill response
+
+  You can [receive the autofill fill request, and send the autofill fill response](#autofill-fill-request-and-autofill-fill-response).
+
+- The storage of autofill data
+
+  You can [receive the request of saving the autofill data](#the-storage-of-autofill-data).
+
+## Prerequisites
+
+1. To use the Autofill Service API (in [mobile](../../api/mobile/latest/group__CAPI__UIX__AUTOFILL__SERVICE_MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__UIX__AUTOFILL__SERVICE_MODULE.html) applications), the application must have meta key in the `tizen-manifest.xml` file to be recognized as an Autofill Service:
```

Comment:
Please change "meta key" to "metadata key"

Reply:
fixed.